src/Spectre.libPeakFinder/ExtremaFinder.cpp

```diff
@@ -16,6 +16,7 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 */
+#include <functional>
```

Comment:
What do we need it for? Is this for `filter`? This `filter` was ours, `spectre::functional::filter` or something like this.

Reply:
I guess that's for `std::less_equal` and `std::greater_equal`

Reply:
That’s precisely the case here.